### ISSUE TITLE
My Rally Event Today (May 30th, 2018)

### ISSUE THREAD

**Issue by DjFIL007** [gRally/dev#616]
Stage 1, no issues
Stage 2, no issues
Stage 3... I complete the stage, says I'd had been third for the stage (via in-game timing)... but as I exit the stage, I get the error.  following crash dump was provided...

[Crash_2018-05-30_164202.zip](https://github.com/gRally/dev/files/2055993/Crash_2018-05-30_164202.zip)

I notice that in-game leaderboards, https://www.grally.net/Live/ (or the bot in the live-rallies discord channel) has not seen my running of Stage 3.

In game Stage 3 is now greyed out and no longer able to run... now about to try and start stage 4...

**Comment by DjFIL007** [comment]
Stage 4 loads correctly, runs with no errors in stage or as I exit stage... while in stage (photo taken as I cross finish) it is showing me times of myself and others...

![grally-ss4](https://user-images.githubusercontent.com/14816160/40753690-37492906-642a-11e8-9daf-046d3e1212e5.png)

the chat bot thinks i ran SS2 again.... this was SS4.  no times showing for SS4 on in-game leaderboards or https://www.grally.net/Live/

now to try Stage 5...

**Comment by DjFIL007** [comment]
Stage 4 loads correctly, runs with no errors in stage or as  I exit stage... while in stage (photo taken as I cross finish) it is  showing me times of myself and others...

![grally-ss5](https://user-images.githubusercontent.com/14816160/40753899-2d16e102-642b-11e8-9714-e7edadad1fa5.png)

Chat bot thinks I ran SS3 again... this was SS5.  no times showing for SS5 on in-game leaderboards or https://www.grally.net/Live/

As I exit the stage, rally has appeared to been closed/removed, as if it was finished normally.  Following new logs were located...

[dump.UnhandledException.txt](https://github.com/gRally/dev/files/2056046/dump.UnhandledException.txt)

[dump.live.txt](https://github.com/gRally/dev/files/2056047/dump.live.txt)

[output_log.txt](https://github.com/gRally/dev/files/2056049/output_log.txt)

**Comment by tsute2** [comment]
I've had a similar situation on today's rally called 24H Test 02. Reselt board on this rally is a little bit odd.

SS1 and 2 were OK but after I finished SS3 then I tried to quit the stage by pressing X button ,I got an error screen and crashlog below.
[Crash_2018-05-31_091704.zip](https://github.com/gRally/dev/files/2056176/Crash_2018-05-31_091704.zip)

I finish SS4 and 5 without problem but it shows no results on SS4 and 5 via gRally client and also www.grally.net despite I've seen several players both stages and discord live rallies bot.
